endpoint/app.py

```diff
@@ -66,12 +66,14 @@ def catchall(path: str):
     if path == "":
         return "Test OK", 200
     endpoint_path = os.getenv("ENDPOINT_PATH")
+
     # Reject requests not matching the one defined in env
-    if endpoint_path != path:
+    if path not in [endpoint_path, "cesva"]:
```

Comment:
What is this hardcoded "cesva" here?

Reply:
I could un-hardcode in a separate patch. We can also discuss whether 'sensornode' is good name for lorawan.